### DISCUSSION TITLE
How to use Shares as public Form

### DISCUSSION THREAD

**Discussion by Lucas-Gruber** [directus/directus#18100] [Archive]
Hey,
is it possible to use the Share-Feature as Google Forms replacement? Like a Bug-Report, Ticket System Survey or anything else?

I would love to use Directus for this, because i have everything else direct in Directus and i hate Google Forms or any other Service like this because i didn't have the full control over the Data and i would also love to replace my companies Ticket System with Directus and this feature would really, really help me.

**Comment by rijkvanzanten** [comment]
Not at present time. The share function currently shares as readonly.

**Reply by w0kyj** [reply]
Just for reference: https://github.com/directus/directus/discussions/18807
